torch_xla/csrc/ir.cpp

```diff
@@ -114,15 +114,11 @@ const xla::Shape& Value::xla_node_shape() const {
 
 Node::Node(torch::lazy::OpKind op, OpList operands, xla::Shape shape,
            size_t num_outputs, torch::lazy::hash_t hash_seed)
-    : torch::lazy::Node(
-          op, num_outputs,
-          /*node_hash=*/torch::lazy::HashCombine(op.hash(), hash_seed),
-          /*dag_hash_fn=*/
-          [&](bool /*bakeInSizes*/) -> torch::lazy::hash_t {
-            return GetOperandHashes(
-                operands, torch::lazy::HashCombine(op.hash(), hash_seed));
-          }),
-      xla_shape_(std::move(shape)) {
+    : torch::lazy::Node(op, num_outputs),
+      xla_shape_(std::move(shape)),
+      node_hash_(torch::lazy::HashCombine(op.hash(), hash_seed)),
+      dag_hash_(GetOperandHashes(
+          operands, torch::lazy::HashCombine(op.hash(), hash_seed))) {
```

Comment:
If we want to minimize computation, I don't think these _need_ to be in the MIL. Doing
```suggestion
      xla_shape_(std::move(shape)) {
      node_hash_ = torch::lazy::HashCombine(op.hash(), hash_seed);
      dag_hash_ = GetOperandHashes(operands, node_hash_);
    }
```
will save one hash calculation per node construction which can add up.

Reply:
oh, nice catch